packages/streamline/src/components/list-items/list-item-option/list-item-option.utils.tsx

```diff
@@ -0,0 +1,76 @@
+import { ColorTheme } from '../../../theme';
+
+interface TextColors {
+  titleColor: ColorTheme;
+  descriptionColor: ColorTheme;
+  headerColor: ColorTheme;
+}
+
+export const getTextColors = ({
+  disabled,
+  selected,
+}: {
+  disabled: boolean;
+  selected: boolean;
+}): TextColors => {
+  if (selected) {
+    return {
+      titleColor: 'BLUKO_500',
+      descriptionColor: 'BLUKO_500',
+      headerColor: 'BLUKO_500',
+    };
+  }
+  if (disabled) {
+    return {
+      titleColor: 'GREY_500',
+      descriptionColor: 'GREY_500',
+      headerColor: 'GREY_500',
+    };
+  }
+  return {
+    titleColor: 'GREY_1000',
+    descriptionColor: 'GREY_700',
+    headerColor: 'GREY_700',
+  };
+};
+
+export const getBackgroundColor = ({
+  pressed,
+  selected,
+  isResolving,
+}: {
+  pressed: boolean;
+  selected: boolean;
+  isResolving: boolean;
+}): ColorTheme => {
+  if (pressed && selected) {
+    return 'BLUKO_50';
+  }
+  if (selected || isResolving) {
+    return 'BLUKO_25';
+  }
+  return 'PURE_WHITE_1000';
+};
+
+export const getBorderColor = ({
+  selected,
+  pressed,
+  disabled,
+  isResolving,
+}: {
+  selected: boolean;
+  disabled: boolean;
+  pressed: boolean;
+  isResolving: boolean;
+}): ColorTheme => {
+  if (disabled) {
+    return 'GREY_100';
+  }
+  if (selected || isResolving) {
+    return 'BLUKO_500';
+  }
+  if (pressed) {
+    return selected ? 'BLUKO_600' : 'GREY_300';
+  }
+  return 'GREY_100';
+};
```

Comment:
You could group these 3 methods into one that will return the 5 colors

Reply:
i find it easier to read than having one big function with a lot of if statement. WDYT